packages/amplify-migration-tests/src/migration-helpers-v10/init.ts

```diff
@@ -0,0 +1,94 @@
+
+import { addCircleCITags, getCLIPath, nspawn as spawn } from '@aws-amplify/amplify-e2e-core';
+import { EOL } from 'os';
+
+const defaultSettings = {
+    name: EOL,
+    // eslint-disable-next-line spellcheck/spell-checker
+    envName: 'integtest',
+    editor: EOL,
+    appType: EOL,
+    framework: EOL,
+    srcDir: EOL,
+    distDir: EOL,
+    buildCmd: EOL,
+    startCmd: EOL,
+    useProfile: EOL,
+    profileName: EOL,
+    region: process.env.CLI_REGION,
+    local: false,
+    disableAmplifyAppCreation: true,
+    disableCIDetection: false,
+    providerConfig: undefined,
+    permissionsBoundaryArn: undefined,
+};
+  
+export function initJSProjectWithProfileV10(cwd: string, settings?: Partial<typeof defaultSettings>): Promise<void> {
```

Comment:
nit: this can be simplified to an async function that returns the Promise from `.runAsync()`

Reply:
similarly here; didn't want to refactor since it is just a snapshot of the e2e-core tooling from v10